### PR TITLE
Added generator for running pod new documentation

### PR DESCRIPTION
**Pull request by jayeshmahajan**

### CODE REVIEW

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -90,12 +90,13 @@ Kubernetes manifests can be defined in YAML or JSON. The file extension `.yaml`,
...
-kubectl apply -f https://git.io/vPieo          # create resource(s) from url
-kubectl create deployment nginx --image=nginx  # start a single instance of nginx
-kubectl explain pods                           # get the documentation for pod manifests
+kubectl apply -f ./my-manifest.yaml                    # create resource(s)
+kubectl apply -f ./my1.yaml -f ./my2.yaml              # create from multiple files
+kubectl apply -f ./dir                                 # create resource(s) in all manifest files in dir
+kubectl apply -f https://git.io/vPieo                  # create resource(s) from url
+kubectl create deployment nginx --image=nginx          # start a single instance of nginx
+kubectl explain pods                                   # get the documentation for pod manifests
+kubectl run --generator=run-pod/v1 nginx --image=nginx # create nginx pod without creating deployment
```
Comment (sftim) [inline content/en/docs/reference/kubectl/cheatsheet.md:99]:
```suggestion
kubectl run --generator=run-pod/v1 nginx --image=nginx # start a single instance of nginx, without creating deployment
```
?

Comment (sftim) [reply content/en/docs/reference/kubectl/cheatsheet.md:99]: Actually, in Kubernetes v1.18 all generators are documented. I don't think it's a good idea to add this to the cheat sheet.

**Comment by sftim** [comment]
@jayeshmahajan generators are deprecated. Are you willing to close this PR?

**Comment by jimangel** [comment]
Closing this PR due to lack of response. Feel free to reopen if desired!
/close

**Pull request closed without merging**